### DISCUSSION TITLE
Can't change the matplotlib settings

### DISCUSSION THREAD

**Discussion by jmzhang1911** [theislab/scvelo#714] [Q&A] [answered]
I use the codes to change the matplotlib settings but it doesn't work. Who can help me solve this problem?
![image](https://user-images.githubusercontent.com/52552382/134925383-6d26fbad-71d7-450c-b164-b1a080f3aff7.png)

**Comment by WeilerP** [comment, marked as answer]
What exactly is not working, *i.e.*, what kind of output are you expecting?

**Reply by jmzhang1911** [reply]
Thank you for your reply, I am sorry for not describing the problem clearly. As shown in the picture below, the first picture is what I got before, and it seems that the color setting is successful, and the second picture is the latest picture I got and it seems that the setting is not successful. How can I solve this problem?
![image](https://user-images.githubusercontent.com/52552382/134932699-d6880a80-97ee-4930-81bd-b517fbe36943.png)

**Reply by WeilerP** [reply]
Could it be that the column containing the cluster assignment in `sample_new` is not categorical?

**Reply by jmzhang1911** [reply]
`sample_old`and`sample_new` are `Categories`. 
I found the `sample.__dict__['_uns']['seurat_clusters_colors']` are different. Is it for this reason? If so, how can I fix it? Thank you
![image](https://user-images.githubusercontent.com/52552382/134938433-81f9fed1-c8e7-43dd-88d0-fc16c11c7b27.png)
![image](https://user-images.githubusercontent.com/52552382/134938574-2fadda40-00e0-4036-bd95-c727e13c7fe0.png)

**Reply by WeilerP** [reply]
Yes, this should be the reason. Just use the same color definitions in both cases, *i.e.*, update one of them to be the same as the other.

**Reply by jmzhang1911** [reply]
Thank you for completely solving my problem, now I can customize the color.🙃🙃🙃
![image](https://user-images.githubusercontent.com/52552382/135011878-aba799c5-3dae-41eb-89d3-cddd6400c49e.png)
